includes/amp/class-captioned-image.php

```diff
@@ -0,0 +1,44 @@
+<?php
+/**
+ * Class Captioned_Image
+ *
+ * @package AMP
+ */
+
+namespace AMP;
+
+/**
+ * Class Captioned_Image
+ *
+ * @internal
+ * @since 1.5.0
+ */
+final class Captioned_Image extends Image implements Has_Caption {
+
+	/**
+	 * The caption text.
+	 *
+	 * @var string
+	 */
+	private $caption;
+
+	/**
+	 * Constructs the class.
+	 *
+	 * @param \DOMElement $image_node The image node.
+	 * @param string      $caption    The caption text.
+	 */
+	public function __construct( \DOMElement $image_node, $caption ) {
```

Comment:
I suggest always importing all classes, even the non-namespaced ones. This will not only get rid of the `\` in front of them, make it more readable, but also will add a clear list of external dependencies at the top of each class:
```php
use DOMElement;
```

This list of dependencies is very valuable later down the road for debugging and refactoring.

```suggestion
	public function __construct( DOMElement $image_node, $caption ) {
```

Reply:
Thanks, https://github.com/ampproject/amp-wp/pull/3659/commits/3f0ee523069fcda4e8588f71d781256dad3346ca imports the classes.